playbooks/deploy-rock.yml

```diff
@@ -619,6 +639,7 @@
       repo: "{{ bro_rockscripts_repo }}"
       dest: /opt/bro/share/bro/site/scripts/rock
       version: "{{ bro_rockscripts_branch }}"
+      #update: no
```

Comment:
Removed. We may want to considering adding this at some point, because local modifications to the scripts will cause the playbook to fail without it. My use case for local modifications was updating the sensor ID.